.github/workflows/run-tests.yml

```diff
@@ -10,9 +10,11 @@ jobs:
             matrix:
                 os: [ubuntu-latest]
                 php: [7.4, 7.3, 7.2]
-                laravel: [7.*, 6.*]
+                laravel: [8.*, 7.*, 6.*]
                 dependency-version: [prefer-lowest, prefer-stable]
                 include:
+                    -   laravel: 8.*
+                        testbench: 6.*
                     -   laravel: 7.*
                         testbench: 5.*
                     -   laravel: 6.*
```

Comment:
The following rule needs to be added as Laravel 8 does no longer support php7.2.
```yaml
exclude:
   -   laravel: 8.*
        php: 7.2
```